prototype_source/pt2e_quant_ptq_static.rst

```diff
@@ -0,0 +1,399 @@
+    # prepared_model = prepare_pt2e(model_to_quantize, {"": qconfig})
+    # loaded_quantized_model = convert_pt2e(prepared_model)
+    # loaded_quantized_model.load_state_dict(torch.load(pt2e_graph_mode_model_file_path))
+
+    # save with script
+    torch.jit.save(torch.jit.script(quantized_model), pt2e_graph_mode_model_file_path)
+    loaded_quantized_model = torch.jit.load(pt2e_graph_mode_model_file_path)
+
+    top1, top5 = evaluate(loaded_quantized_model, criterion, data_loader_test)
+    print("[after serialization/deserialization] Evaluation accuracy on test dataset: %2.2f, %2.2f"%(top1.avg, top5.avg))
+
+If you want to get better accuracy or performance,  try configure `quantizer` in different ways.
+
+9. Debugging Quantized Model
+----------------------------
+We have Numeric Suite that can help with debugging in eager mode and fx graph mode, the new version of Numeric Suite working with PyTorch 2.0 Export models is still in development.
+
+10. Comparison with Baseline Float Model and Eager Mode Quantization
+--------------------------------------------------------------------
+
+.. code:: python
+
+    scripted_float_model_file = "resnet18_scripted.pth"
+
+    print("Size of baseline model")
+    print_size_of_model(float_model)
+
+    top1, top5 = evaluate(float_model, criterion, data_loader_test)
+    print("Baseline Float Model Evaluation accuracy: %2.2f, %2.2f"%(top1.avg, top5.avg))
+    torch.jit.save(torch.jit.script(float_model), saved_model_dir + scripted_float_model_file)
+
+In this section, we compare the model quantized with PT2 Export quantization with the model
+quantized in eager mode. PT2 Export quantization and eager mode quantization produce very similar quantized models,
+so the expectation is that the accuracy and speedup are similar as well.
+
+.. code:: python
+
+    print("Size of PT2 Export quantized model")
+    print_size_of_model(quantized_model)
+    top1, top5 = evaluate(quantized_model, criterion, data_loader_test)
+    print("PT2 Export quantized model Evaluation accuracy on test dataset: %2.2f, %2.2f"%(top1.avg, top5.avg))
+
+    from torchvision.models.quantization.resnet import resnet18
+    eager_quantized_model = resnet18(pretrained=True, quantize=True).eval()
+    print("Size of eager mode quantized model")
+    eager_quantized_model = torch.jit.script(eager_quantized_model)
+    print_size_of_model(eager_quantized_model)
+    top1, top5 = evaluate(eager_quantized_model, criterion, data_loader_test)
+    print("eager mode quantized model Evaluation accuracy on test dataset: %2.2f, %2.2f"%(top1.avg, top5.avg))
+    eager_mode_model_file = "resnet18_eager_mode_quantized.pth"
+    torch.jit.save(eager_quantized_model, saved_model_dir + eager_mode_model_file)
+
+We can see that the model size and accuracy of  graph mode and eager mode quantized model are pretty similar.
+
+Running the model in AIBench (with single threading) gives the following result:
+
+(TODO): update numbers
```

Comment:
```suggestion
```

Reply:
If you want to keep the todo, add as a comment so it's not visible in the final HTML: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#comments